.github/workflows/build-package.yml

```
@@ -0,0 +1,52 @@
+name: Build Package
+
+env:
+  GH_TOKEN: ${{ secrets.GH_TOKEN }}
+  PACKAGING_PASSPHRASE: ${{ secrets.PACKAGING_PASSPHRASE }}
+  PACKAGING_SECRET_KEY: ${{ secrets.PACKAGING_SECRET_KEY }}
+
+on:
+  push:
+    branches: "**"
+
+  workflow_dispatch:
+
+jobs:
+  build_package:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        platform: [ el/7, el/8, ol/7, debian/stretch, debian/buster, ubuntu/xenial, ubuntu/bionic,ubuntu/focal ]
+#        platform: [ debian/stretch ]
+    env:
+      PLATFORM: ${{ matrix.platform }}
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      #      - name: install dependencies
+      #        run: sudo apt install libcurl4-openssl-dev libssl-dev
+      #
+      #      - name: Define git credentails
+      #        run: git config --global user.email "gindibay@microsoft.com"&& git config --global user.name "Gurkan Indibay"
+
+      - name: Clone tools branch
+        run: git clone --depth=1 --branch tools-transition-update-docker https://github.com/citusdata/tools.git tools
+
+      - name: Install package dependencies
+        run: sudo apt install libcurl4-openssl-dev libssl-dev
+
+      - name: Install python requirements
+        run: python -m pip install -r tools/packaging_automation/requirements.txt
+
+      - name: Build Packages
+        run: |
+          python -m  tools.packaging_automation.citus_package --gh_token "${{ secrets.GH_TOKEN }}" --platform "${{ matrix.platform }}" \
+          --build_type 'release' --secret_key "${{ secrets.PACKAGING_SECRET_KEY }}" --passphrase "${{ secrets.PACKAGING_PASSPHRASE }}" \
+          --output_dir "$(pwd)/packages/" --input_files_dir "$(pwd)" --output_validation "False"
+
+      - name: Publish Packages
+        run: |
+          python -m  tools.packaging_automation.upload_to_package_cloud --platform ${{ matrix.platform }} --package_cloud_api_token ${{ secrets.PACKAGE_CLOUD_API_TOKEN }} \
+          --repository_name "sample" --output_file_path "$(pwd)/packages"
```

I do not see a secret for `PACKAGE_CLOUD_API_TOKEN` in the env section. How does this actually work?

Also this is hard to read for me.

```suggestion
          python -m  tools.packaging_automation.upload_to_package_cloud \
                 --platform ${{ matrix.platform }} \
                 --package_cloud_api_token ${{ secrets.PACKAGE_CLOUD_API_TOKEN }} \
                 --repository_name "sample" \
                 --output_file_path "$(pwd)/packages"
```

---

Fixed